test/conversion_test.go

```diff
@@ -1184,9 +1186,19 @@ func TestBundleConversion(t *testing.T) {
 
 	v1TaskRunExpected.Status.Provenance = &v1.Provenance{
 		FeatureFlags: getFeatureFlagsBaseOnAPIFlag(t),
+		RefSource: &v1.RefSource{
```

Comment:
I think provenance is updated when we are using the resolver to fetch the bundle from the registry that was setup in the `setupBundle` helper?

And for this i had 2 options:
- to refactor the helper func to return the image digest
- to ignore the image digest

which i have chosen the 2nd one since this test is mainly for conversions if that makes sense?